src/sbvr-api/odata-response.ts

```diff
@@ -158,9 +173,17 @@ export const process = async (
 	);
 
 	const odataIdField = sqlNameToODataName(table.idField);
+	const fetchProcessingPromises: Array<Promise<void>> = [];
 	for (const row of rows) {
 		for (const fieldName of processedFields) {
 			row[fieldName] = fetchProcessingFields[fieldName](row[fieldName]);
+			if (row[fieldName] instanceof Promise) {
```

Comment:
@fisehara  We also could just do `await` on the fetchProcessingFields as the presigning promises should be really fast because of the caching on the signing but I think it is very weird to have nested for loop with await inside, so I optmised with `Promise.all` the impact for anything that is not a webresource is basically a if check on instance that I would expect to be fairly fast